ui/App/UserProfileScreen.svelte

```diff
@@ -30,14 +43,105 @@
       },
     });
   }
+
+  let registration: Registration | undefined;
+  let ownedOrgs: Org[] = [];
+
+  const wallet = svelteStore.get(Wallet.store);
+  const ethereumEnvironment = ethereum.selectedEnvironment;
+
+  async function loadSidebarData(): Promise<void> {
```

Comment:
```suggestion
  async function loadSidebarData(): Promise<Org[]> {
```

And do the assignment where the function is called.

Reply:
it's assigning both `registration` as well as `orgs`. And I had trouble doing that in `ProfileScreen`, if you tell me what I did wrong there, I might be able to do it here.

Reply:
Nah, let's leave it as is. I didn't realise it was async. This would make the call more complicated.